IdentityCore/src/webview/response/MSIDWebWPJResponse.m

```diff
@@ -27,9 +27,16 @@
 
 #import "MSIDWebWPJResponse.h"
 #import "MSIDClientInfo.h"
+#import "MSIDBrokerConstants.h"
+#import "MSIDOperationFactory.h"
+#import "MSIDBrokerInstallOperation.h"
 
 @implementation MSIDWebWPJResponse
 
++ (void)load {
+    [MSIDOperationFactory registerOperationClass:MSIDBrokerInstallOperation.class forResponseClass:self];
```

Comment:
If there's 1-1 mapping between web response and the operation, can we simplify and let response create the operation instead of having an additional factory?

Reply:
Not for this task, but for future extension there should be different operation even for the same response.